### ISSUE TITLE
jlink-image already completed error on Windows

### ISSUE THREAD

**Issue by batsatt** [oracle/helidon-build-tools#322]
```
[ERROR] Failed to execute goal io.helidon.build-tools:helidon-maven-plugin:2.1.2:jlink-image (jlink-image) on project quickstart-mp: Image generation error: java.lang.IllegalStateException: already completed -> [Help 1]

...

Caused by: java.lang.IllegalStateException: already completed

    at io.helidon.build.util.ProcessMonitor.assertRunning (ProcessMonitor.java:494)
    at io.helidon.build.util.ProcessMonitor.stdErr (ProcessMonitor.java:398)
    at io.helidon.linker.StartScript.checkWindowsExecutionPolicyError (StartScript.java:160)
    at io.helidon.linker.StartScript.execute (StartScript.java:145)
```

**Comment by batsatt** [comment]
Fixed in #319.

**Comment by batsatt** [comment]
@m0mus FYI
